var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -16,23 +16,24 @@ class Hpx(CMakePackage, CudaPackage, ROCmPackage):
     homepage = "https://hpx.stellar-group.org/"
     url = "https://github.com/STEllAR-GROUP/hpx/archive/1.2.1.tar.gz"
     git = "https://github.com/STEllAR-GROUP/hpx.git"
-    maintainers = ['msimberg', 'albestro', 'teonnik']
+    maintainers = ['msimberg', 'albestro', 'teonnik', 'hkaiser']
 
     tags = ['e4s']
 
     version('master', branch='master')
     version('stable', tag='stable')
+    version('1.8.0', sha256='93f147ab7cf0ab4161f37680ea720d3baeb86540a95382f2fb591645b2a9b135')
     version('1.7.1', sha256='008a0335def3c551cba31452eda035d7e914e3e4f77eec679eea070ac71bd83b')
     version('1.7.0', sha256='05099b860410aa5d8a10d6915b1a8818733aa1aa2d5f2b9774730ca7e6de5fac')
-    version('1.6.0', sha256='4ab715613c1e1808edc93451781cc9bc98feec4e422ccd4322858a680f6d9017')
+    version('1.6.0', sha256='4ab715613c1e1808edc93451781cc9bc98feec4e422ccd4322858a680f6d9017', deprecated=True)
```

Comment:
In what sense is 1.6.0 and older deprecated, but 1.7.X not?

Reply:
I'm still not sure I understand the logic behind which ones are deprecated and which not. Is the idea to deprecate all versions except the latest patch release for each minor release? If yes, shouldn't 1.6.0 stay undeprecated in that case (and 1.1.0 as well)? Or is there some other logic to it?

Reply:
Fair points. Apparently, I messed it up. My initial idea was to have only the latest versions of a particular release non-deprecated. I'm not sure what the usual way of marking releases as deprecated is applied for Spack packages.

Reply:
As far as I understand it `deprecated=True` means that that version is about to removed from spack, not that that version is e.g. no longer supported or something similar upstream.

Reply:
Ok, I'll remove the deprecated=True, then. Thanks!